frontend/common/stores/feature-list-store.js

```diff
@@ -196,6 +197,7 @@ const controller = {
     onComplete,
   ) => {
     let prom
+    const featureFlagId = flag.id || projectFlag.id || environmentFlag.feature
```

Comment:
@kyle-ssg  Is this is a good way to determine the feature flag ID?

Reply:
projectFlag.id

Reply:
Thanks @kyle-ssg , I am using now the `v.feature` value which was the one used before my changes. I am using the same as the future inside the `feature_segment_value` structure.